.github/workflows/unittests.yml

```diff
@@ -1,57 +1,60 @@
-name: Unit Tests
-on:
-  push:
-    branches:
-      - "*"
-  pull_request:
-    branches:
-      - "*"
-  schedule:
-    - cron: "0 0 * * 0"
+ name: Unit Tests
+ on:
+   push:
+     branches:
+     - '*'
+   pull_request:
+     branches:
+     - '*'
+   schedule:
+     - cron: '59 23 * * *'
 
-jobs:
-  unittests:
-    name: CI (${{ matrix.os }}-${{ matrix.environment-file }})
-    runs-on: ${{ matrix.os }}
-    continue-on-error: true
-    timeout-minutes: 45
-    strategy:
-      matrix:
-        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        environment-file: [.ci/37.yml, .ci/38.yml, .ci/39.yml]
-        fail-fast: [false]
-    defaults:
-      run:
-        shell: bash -l {0}
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
-        env:
-          CACHE_NUMBER: 0
-        with:
-          path: ~/conda_pkgs_dir
-          key: ${{ matrix.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles(matrix.environment-file) }}
-      - uses: conda-incubator/setup-miniconda@v2
-        with:
-          miniconda-version: "latest"
-          mamba-version: "*"
-          channels: conda-forge
-          channel-priority: true
-          auto-update-conda: false
-          auto-activate-base: false
-          environment-file: ${{ matrix.environment-file }}
-          activate-environment: test
-          use-only-tar-bz2: true
-      - run: conda info --all
-      - run: conda list
-      - run: conda config --show-sources
-      - run: conda config --show
-      - run: pip install -e . --no-deps --force-reinstall
-      - name: Pytest
-        run: py.test -v tobler --cov=tobler --cov-report=xml
-      - name: codecov (${{ matrix.os }}, ${{ matrix.environment-file }})
-        uses: codecov/codecov-action@v1
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          file: ./coverage.xml
-          name: tobler-codecov
+ jobs:
+   unittests:
+     name: CI (${{ matrix.os }}-${{ matrix.environment-file }})
+     runs-on: ${{ matrix.os }}
+     continue-on-error: false
+     timeout-minutes: 45
+     strategy:
+       matrix:
+         os: [ubuntu-latest, macos-latest, windows-latest]
+         environment-file: [ .ci/39.yml, .ci/310.yml, .ci/311.yml]
+         experimental: [false]
+         fail-fast: [false]
```

Comment:
Seems like the `fail-fast` flag is [not working properly](https://github.com/pysal/tobler/actions/runs/4928566734/jobs/8807090971?pr=174#step:7:131). Try removing the brackets around both `experimental` and `fail-fast` since these aren't iterables.

Reply:
they need to be iterables to work through the matrix (so removing the brackets will break) but i think i can move it up out of the matrix to have it apply globally. Let me try that

Reply:
the gha verifier doesnt like them in either place, actually. I dunno i'll keep poking 
<img width="646" alt="Screenshot 2023-05-09 at 4 53 45 PM" src="https://github.com/pysal/tobler/assets/4213368/06b657dc-5d8b-44ff-bf2b-e28d941d0853">

Reply:
Check out [`spopt`](https://github.com/pysal/spopt/blob/main/.github/workflows/testing.yml#L35). Same level as `matrix`.

Reply:
nice, thanks

Reply:
Let's see how that works and if maybe only Windows 3.9 is failing.